fix_includes_test.py

```diff
@@ -33,8 +33,9 @@
 class FakeFlags(object):
   def __init__(self):
     self.blank_lines = False
-    self.comments = True
-    self.update_comments = False
+    self.comments = "short"
+    self.nocomments = False
```

Comment:
This seems redundant since `comments` is already `short`. I don't think `nocomments` is used directly, right?